autosklearn/data/validation.py

```diff
@@ -406,10 +406,19 @@ def _check_and_encode_target(
 
             # Mypy redefinition
             assert self.target_encoder is not None
-            self.target_encoder.fit(y)
+
+            # remove ravel warning from pandas Series
+            if len(y.shape) > 1 and y.shape[1] == 1 and hasattr(y, "values"):
+                self.target_encoder.fit(y.values.ravel())
```

Comment:
Could you please use 'to_numpy' here instead of 'values'? By that it is clear what the return value is, see [here](https://pandas.pydata.org/pandas-docs/stable/whatsnew/v0.24.0.html#accessing-the-values-in-a-series-or-index).

Reply:
I see, thanks I will move to this